src/p2p/pubsub.rs

```diff
@@ -206,10 +216,17 @@ impl Pubsub {
     }
 
     /// Returns the peers known to subscribe to the given topic
-    pub fn subscribed_peers(&self, topic: &Topic) -> Vec<PeerId> {
+    pub fn subscribed_peers(&self, topic: &str) -> Vec<PeerId> {
```

Comment:
Not sure but maybe we could use `Gossipsub::mesh_peers` for this in some way since the protocol does keep a mesh peers for a specific topic hash.